src/plugins/intel_cpu/src/transformations/cpu_opset/x64/pass/qkv_proj_fusion.cpp

```diff
@@ -52,6 +52,10 @@ ov::intel_cpu::QKVProjFusion::QKVProjFusion() {
             return false;
         }
 
+        if (!src.get_element_type().is_real()) {
```

Comment:
In general such limitations (caused by lack of kernel support) should be proccessed via callback mechanism.